search-resources-api/src/main/java/no/unit/nva/search/SearchTicketsHandler.java

```diff
@@ -39,153 +39,166 @@
-  //TODO: When the Cristin proxy is mature and quick, we should query the Cristin proxy in
-  // order to avoid using semantically charged identifiers.
-  private ViewingScope authorizeCustomViewingScope(ViewingScope viewingScope, RequestInfo requestInfo)
-      throws ForbiddenException {
-    var customerCristinId = requestInfo.getTopLevelOrgCristinId().orElseThrow();
-    logger.info("customerCristinId: {}", customerCristinId);
-    return userIsAuthorized(viewingScope, customerCristinId);
-  }
-
-  private Try<ViewingScope> defaultViewingScope(RequestInfo requestInfo) {
-    return attempt(requestInfo::getUserName)
-               .map(nvaUsername -> identityClient.getUser(nvaUsername, requestInfo.getAuthHeader()))
-               .map(Optional::orElseThrow)
-               .map(UserResponse::getViewingScope);
-  }
-
-  private Optional<ViewingScope> getUserDefinedViewingScore(RequestInfo requestInfo) {
-    return requestInfo.getQueryParameterOpt(VIEWING_SCOPE_QUERY_PARAMETER)
-               .map(URI::create)
-               .map(ViewingScope::create);
-  }
-
-  private ViewingScope userIsAuthorized(ViewingScope viewingScope, URI customerCristinId) throws ForbiddenException {
-    if (allIncludedUnitsAreLegal(viewingScope, customerCristinId)) {
-      return viewingScope;
-    }
-    throw new ForbiddenException();
-  }
-
-  private boolean allIncludedUnitsAreLegal(ViewingScope viewingScope, URI customerCristinId) {
-    return viewingScope.getIncludedUnits().stream()
-               .map(requestedOrg -> isUnderUsersInstitution(requestedOrg, customerCristinId))
-               .allMatch(isEqual(true));
-  }
-
-  private boolean isUnderUsersInstitution(URI requestedOrg, URI customerCristinId) {
-    logger.info("viewingScope for requeest institution: {}", requestedOrg);
-    logger.info("viewingScope for institution: {}", customerCristinId);
-    String requestedOrgInstitutionNumber = extractInstitutionNumberFromRequestedOrganization(requestedOrg);
-    String customerCristinInstitutionNumber = extractInstitutionNumberFromRequestedOrganization(customerCristinId);
-    return customerCristinInstitutionNumber.equals(requestedOrgInstitutionNumber);
-  }
-
-  private String extractInstitutionNumberFromRequestedOrganization(URI requestedOrg) {
-    String requestedOrgCristinIdentifier = UriWrapper.fromUri(requestedOrg).getLastPathElement();
-    return requestedOrgCristinIdentifier.split(CRISTIN_ORG_LEVEL_DELIMITER)[HIGHEST_LEVEL_ORGANIZATION];
-  }
-
-  private String getIndexName(RequestInfo requestInfo) {
-    String requestPath = RequestUtil.getRequestPath(requestInfo);
-    return UnixPath.of(requestPath).getLastPathElement();
-  }
+    public static final String VIEWING_SCOPE_QUERY_PARAMETER = "viewingScope";
+    public static final String CRISTIN_ORG_LEVEL_DELIMITER = "\\.";
+    public static final int HIGHEST_LEVEL_ORGANIZATION = 0;
+    public static final String ACCESS_RIGHTS_TO_VIEW_TICKETS = "APPROVE_DOI_REQUEST";
```

Comment:
Her har vi kokt ut rollen "kurator" for access_right Approve doi request. Vet ikke helt om jeg synes dette er design pris, men la gå.